specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlCommon.json

```diff
@@ -0,0 +1,95 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Database Migration Service Resource Provider",
+    "version": "2022-11-30-preview"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "MySqlBinlogPositionInput": {
+      "type": "object",
+      "description": "MySQL binlog reference",
+      "properties": {
+        "filename": {
+          "type": "string",
+          "description": "Specifies the MySQL binlog file name"
+        },
+        "position": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Specifies the position in the binlog file"
+        }
+      }
+    },
+    "MySqlBinlogPositionOutput": {
+      "type": "object",
+      "description": "MySQL binlog reference",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Specifies the MySQL binlog file name",
+          "readOnly": true
+        },
+        "position": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Specifies the position in the binlog file",
+          "readOnly": true
+        },
+        "fileSeqNum": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Specifies the the sequential number of the binlog fie, extracted from the file name",
+          "readOnly": true
+        }
+      }
+    },
+    "MySqlSchemaMigrationOptions": {
+      "type": "object",
+      "description": "MySQL schema migration options",
+      "properties": {
+        "migrateAllViews": {
```

Comment:
>migrateAllViews [](http://example.com/codeflow?start=9&length=15)

Same name and type feedback for these properties as for similarly-named properties of MigrateMySqlAzureDbForMySqlOfflineTaskInput.